docs/api_reference/src/quantities.tex

```diff
@@ -863,66 +857,126 @@
 \item
-Otherwise,
-TBD.
+\tcode{Pred} is a total order for the mapped types.
 \end{itemize}
-\end{itemdescr}
-
-\begin{itemdecl}
-template<template<typename...> typename To, typename OneType,
-         template<typename, typename> typename Pred = @\exposidnc{type-less}@, typename Lhs, typename Rhs>
-consteval auto @\exposidnc{expr-divide}@(Lhs lhs, Rhs rhs);
-\end{itemdecl}
 
-\begin{itemdescr}
 \pnum
-\exposidnc{expr-divide} divides two sorted expression templates.
-
+\effects
 \pnum
-\returns
-TBD.
-\end{itemdescr}
-
-\begin{itemdecl}
-template<template<typename...> typename To, typename OneType, typename T>
-consteval auto @\exposidnc{expr-invert}@(T);
-\end{itemdecl}
+First, inputs to the operations are obtained from the types of the function parameters.
+If the type of a function parameter is:
+\begin{itemize}
+\item
+A specialization of \tcode{To},
+then its template arguments are the terms of a product $x$,
+its input value is $x$, and
+the following also apply.
+\item
+A specialization of \tcode{per},
+then its template arguments are the terms of a product $x$,
+its input value is $1/x$, and
+the following also apply.
+\item
+A specialization of \tcode{power<$x$, $n$>}, then its input value is $x^n$, and
+a specialization of \tcode{power<$x$, $n$, $d$>}, then its input value is $x^{n/d}$, and
+the following also apply.
+\item
+Otherwise, a type $x$ is an input constant.
+\end{itemize}
+\begin{example}
+The input of \tcode{derived_unit<si::kilo_<si::metre>, per<power<non_si::hour, 2>>>}
+is $\tcode{km}/\tcode{h}^2$.
+\end{example}
 
-\begin{itemdescr}
 \pnum
-\exposidnc{expr-invert} inverts the expression template \tcode{T}.
+Then, the operation takes place:
+\begin{itemize}
+\item
+\exposid{expr-multiply} multiplies its inputs,
+\item
+\exposid{expr-divide} divides the inputs of its first parameter by the inputs of its second parameter,
+\item
+\exposid{expr-invert} divides $1$ by its inputs, and
+\item
+\exposid{expr-pow} raises its inputs to the $\tcode{Num}/\tcode{Den}$.
+\end{itemize}
 
 \pnum
-\returns
-TBD.
-\end{itemdescr}
-
-\begin{itemdecl}
-template<std::intmax_t Num, std::intmax_t Den, template<typename...> typename To,
-         typename OneType, template<typename, typename> typename Pred = @\exposidnc{type-less}@, typename T>
-  requires(Den != 0)
-consteval auto @\exposidnc{expr-pow}@(T);
-\end{itemdecl}
+Finally, the result is simplified:
+\begin{itemize}
+\item
+There is at most a single fraction of constants.
+\item
+There is at most a single term with a given constant.
+\item
+$x/1$ is simplified to $x$.
+\item
+$x^{-n}$ is simplified to $1/x^n$, and
+$1/x^{-n}$ is simplified to $x^n$.
+\end{itemize}
+\begin{example}
+$\tcode{m}^{2/1} = \tcode{m}^2$, $\tcode{m}^{-1} = 1/\tcode{m}$.
+\end{example}
 
-\begin{itemdescr}
 \pnum
-\exposidnc{expr-pow} computes the power\irefiev{102-02-08} $\tcode{T}^{\tcode{Num}/\tcode{Den}}$.
+\returns
+The result $y$ of the operation in the \Fundescx{Effects} is mapped to the return type:
```

Comment:
It is not clearly defined what `y` is. Also, `x` is hard to deduce from the "First" part.

Reply:
All the $x$ are introduced local to their item.
This $y$ is local to this list.